spec/lib/audit/checksum_validator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+require_relative "../../../lib/audit/checksum_validator.rb"
+# FIXME: remove this rubocop once we start writing tests
+# rubocop:disable RSpec/RepeatedExample
+# TODO: implement this;  we begin with a placeholder
+
+RSpec.describe ChecksumValidator do
+
+  describe ".checksum_validate_disk" do
+    described_class.checksum_validate_disk('2018-02-05 21:37:23 UTC', 'services-disk04', 'MD5')
+    skip 'we should figure out what they are and test them'
+
+  end
+
+  describe ".checksum_validate_disk_profiled" do
+    it "spins up a profiler, calling profiling and printing methods on it" do
+      described_class.checksum_validate_disk_profiled('2018-02-05 21:37:23 UTC', 'services-disk04', 'MD5')
+      skip 'we should figure out what they are and test them'
+    end
```

Comment:
I think you can actually write the "profiled" flavor tests now, if you look at C2M and M2C ...

In fact, you could have TWO tests (and perhaps C2M and M2C should have analogous 2 tests in the context for each profiled method)

```ruby
it "spins up a profiler, calling profiling and printing methods on it" do
      mock_profiler = instance_double(Profiler)
      expect(Profiler).to receive(:new).and_return(mock_profiler)
      expect(mock_profiler).to receive(:prof)
      expect(mock_profiler).to receive(:print_results_flat).with('C2M_check_version_on_dir')
      subject
    end

it "calls .checksum_validate_disk" do
      mock_profiler = instance_double(Profiler)
      allow(Profiler).to receive(:new).and_return(mock_profiler)
      allow(mock_profiler).to receive(:prof)
      allow(mock_profiler).to receive(:print_results_flat).with('C2M_check_version_on_dir')
      expect(ChecksumValidator).to receive(:checksum_validate_disk)
      subject
    end
```